modules/cluster/src/main/java/io/fluo/cluster/runner/YarnAppRunner.java

```diff
@@ -176,23 +223,23 @@ public void start() {
     try {
       // set twill run id zookeeper
       String twillId = controller.getRunId().toString();
-      CuratorUtil.putData(getCurator(), ZookeeperPath.YARN_TWILL_ID,
-          twillId.getBytes(StandardCharsets.UTF_8), CuratorUtil.NodeExistsPolicy.FAIL);
+      CuratorUtil.putData(getAppCurator(config), ZookeeperPath.YARN_TWILL_ID,
+          twillId.getBytes(StandardCharsets.UTF_8), CuratorUtil.NodeExistsPolicy.OVERWRITE);
```

Comment:
Running `fluo stop <app>` used to delete these ZK nodes so they would never exist if you restart fluo.   I removed the code that deleted these nodes so `fluo list` could print state and the YARN app ID.